app/views/peer_reviews/_peer_reviews_new_manager.js.jsx.erb

```diff
@@ -1,6 +1,9 @@
 <script type='text/jsx'>
   /** @jsx React.DOM */
 
+  // A constant that represents all sections
+  var ALL_SECTIONS = '';
```

Comment:
I set this to be a constant that is used to identify an "all selections selected" from the dropdown menu, since the dropdown menu has a non-empty string for the section name being selected. This would require a change to the `Section` model where:

```rails
validates_presence_of :name
```

would become

``rails
validates :name, presence: true, allow_blank: false
```

So that way this would be correct. The form may prevent an empty section name from being set when creating them though.